Source/WebCore/platform/ScrollAnimator.cpp

```diff
@@ -182,6 +182,13 @@ bool ScrollAnimator::handleWheelEvent(const PlatformWheelEvent& wheelEvent)
 // "Stepped scrolling" is only used by RenderListBox. It's special in that it has no rubberbanding, and scroll deltas respect Scrollbar::pixelStep().
 bool ScrollAnimator::handleSteppedScrolling(const PlatformWheelEvent& wheelEvent)
 {
+#if PLATFORM(MAC)
```

Comment:
Should these be `#if ENABLE(KINETIC_SCROLLING)` instead? Is this really macOS only?